### ISSUE TITLE
Error on iOS 12

### ISSUE THREAD

**Issue by rafaelkapela** [willmcpo/body-scroll-lock#39]
disableBodyScroll(targetElement) doesn't work :(

**Comment by equinusocio** [comment]
It works on my iPhone X with iOS 12..

**Comment by willmcpo** [comment]
what is the error? Screenshots?

**Comment by yannickoo** [comment]
I also couldn't reproduce that one on 📱 with latest iOS.

**Comment by knanduri** [comment]
Works for me too on iphone X with IOS12

**Comment by MarsCaiWORD** [comment]
me to , but no errors

**Comment by willmcpo** [comment]
For those experiencing this problem, it would be helpful if I could get some assistance in diagnosing the issue as I don't currently have an iOS 12 device that is having this issue.

**Comment by mark-bucknell** [comment]
It works until someone does a two finger scroll

**Comment by kokers** [comment]
Yes, there is a two finger scroll issue, for iOS, but it's also for 11.x versions. Not only for 12. Simply use two fingers and locked content will be all over the place.

**Comment by willmcpo** [comment]
There was a recent change that made the assumption that if there are 2 fingers used, take it as if it was a *pinching* gesture. See https://github.com/willmcpo/body-scroll-lock/pull/35.

Can't cater for all unfortunately - perhaps we should allow the pinching/zoom functionality opt-in ?

I'd be happy to review a PR for this - won't be able to assist with a fix until a little later.

**Comment by kokers** [comment]
That might be it. I think that making this as an opt-in option would be great solution, gonna try to cook something.

**Comment by willmcpo** [comment]
If anyone is able to submit a PR, that'd be great. Sorry I can't get to this in the short term (babies kids etc). 😦
